charcoal-autocomments/comments.js

```diff
@@ -3,35 +3,35 @@ callback(
   [
     {
       name: "[Q,A] Vandalism",
-      description: "Please do not vandalise your posts. Once you have submitted a post, you have licensed the content to the $SITENAME$ community at large ([under the CC BY-SA license](https://creativecommons.org/licenses/by-sa/3.0/)). By SE policy, any vandalism will be reverted."
+      description: "Please do not vandalize your posts. Once you have submitted a post, you have licensed the content to the $SITENAME$ community at large ([under the CC BY-SA license](https://creativecommons.org/licenses/by-sa/4.0/)). By SE policy, any vandalism will be reverted."
```

Comment:
While I'm more familiar with "vandalize" than "vandalise", I'm not really sure we should be switching back and forth between US and British versions of the same word.

Reply:
Ah, British spellings...it was flagged as an incorrect spelling in my spell checker, so this got "fixed" too. Yeah, you can edit this back (I left the "allow maintainers to edit" box checked).